### PR TITLE
Updated nginx.

### PR DESCRIPTION
**Pull request by thresheek**

### CODE REVIEW

**Comment by psftw** [comment]
Hmm, what about the 1.7.10 release?  Someone was asking about it a few days ago in #docker-library :icecream:

**Comment by yosifkit** [comment]
@thresheek, looks like the repo was updated to `1.7.10` but you forgot to bump the number in the library file here.

**Comment by tianon** [comment]
@thresheek we don't get notified when you update your PR, so feel free to ping us next time :+1:

**Comment by tianon** [comment]
LGTM -- @yosifkit can you do the build test?

File: library/nginx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # maintainer: NGINX Docker Maintainers <docker-maint@nginx.com> (@nginxinc)
 
-latest: git://github.com/nginxinc/docker-nginx@57da11369acbec3256b0c2704a50282eeabb684f
-1: git://github.com/nginxinc/docker-nginx@57da11369acbec3256b0c2704a50282eeabb684f
-1.7: git://github.com/nginxinc/docker-nginx@57da11369acbec3256b0c2704a50282eeabb684f
-1.7.9: git://github.com/nginxinc/docker-nginx@57da11369acbec3256b0c2704a50282eeabb684f
+latest: git://github.com/nginxinc/docker-nginx@e35d3022416ca5221fcf958ed8aa6a55a5b043af
+1: git://github.com/nginxinc/docker-nginx@e35d3022416ca5221fcf958ed8aa6a55a5b043af
+1.7: git://github.com/nginxinc/docker-nginx@e35d3022416ca5221fcf958ed8aa6a55a5b043af
+1.7.9: git://github.com/nginxinc/docker-nginx@e35d3022416ca5221fcf958ed8aa6a55a5b043af
```
Comment (yosifkit) [inline library/nginx]: Shouldn't this be 1.7.10 like the Dockerfile at that commit?

**Comment by thresheek** [comment]
Hi, I've updated the version now. Thank you!

**Comment by yosifkit** [comment]
LGTM
Build test of #494; 66804e4b2edeebaf27cb46f0d775717720610837 (`nginx`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/66804e4b2edeebaf27cb46f0d775717720610837/library/nginx"
$ bashbrew build "$url"
Processing nginx:latest ...
Processing nginx:1 ...
Processing nginx:1.7 ...
Processing nginx:1.7.10 ...
$ bashbrew list "$url" | xargs test/run.sh
testing nginx:latest
	'utc' [1/2]...passed
	'cve-2014--shellshock' [2/2]...passed
testing nginx:1
	'utc' [1/2]...passed
	'cve-2014--shellshock' [2/2]...passed
testing nginx:1.7
	'utc' [1/2]...passed
	'cve-2014--shellshock' [2/2]...passed
testing nginx:1.7.10
	'utc' [1/2]...passed
	'cve-2014--shellshock' [2/2]...passed
```

**Pull request merged**